src/web/dataadapters/GreyNoiseCommunityIpLookupAdapterDocumentation.jsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+/* eslint-disable react/no-unescaped-entities, no-template-curly-in-string */
+import React from 'react';
+
+import { ExternalLink } from 'components/common';
+
+class GreyNoiseCommunityIpLookupAdapterDocumentation extends React.Component {
```

Comment:
Do you think there is any value is adding an example lookup response in the documentation? Would that help the user to see what response format to expect? What do you think @cobaltclaudia @waab76 @cobaltclaudia 

A similar example of documentation from the DNS Lookup adapter:
![image](https://user-images.githubusercontent.com/3423655/140816313-4443ab19-9cfe-4a17-aba7-288517e722b1.png)